locales/po/cacti.pot

```diff
@@ -0,0 +1,218 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
This file should probably be quicktree.pot rather than cacti.pot since it's a different repository.

Reply:
I don't think that is important really.

Reply:
It won't be if it gets transferred over. :)